src/Chapter1.hs

```diff
@@ -641,7 +655,13 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n = go (abs n)
+  where
+    go :: Int -> Int
+    go n'
+        | n' >= 10 = go (div n' 10)
+        | otherwise = n'
```

Comment:
Nice solution using the Recursive go pattern! And very well done, that you call `abs` only once :trophy: